src/CoreBundle/private/rendering/layouts/includes/cms_head_data.inc.php

```diff
@@ -32,10 +34,9 @@
 <script src="<?= TGlobal::GetStaticURLToWebLib('/bootstrap/js/bootstrap.bundle.min.js?v4.1.3'); ?>" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->
 <script src="<?= TGlobal::GetStaticURLToWebLib('/components/bootstrap3-typeahead/bootstrap3-typeahead.min.js'); ?>" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->
 <script src="<?= TGlobal::GetStaticURLToWebLib('/javascript/jquery/jquery-form-4.2.2/jquery.form.min.js'); ?>" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->
-<script src="<?= TGlobal::GetStaticURLToWebLib('/javascript/cms.js'); ?>" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->
```

Comment:
looks to me as if we load cms.js via resource collection - but none of the other libraries. Why the special handling of cms.js?

Reply:
all of these are similar - just the location of cms.js changed

Reply:
cms.js is more or less the "base" library for the backend and could be used/required by others: so it should be among the first (after jquery)